src/store/actions/weather.js

```diff
@@ -0,0 +1,10 @@
+import axios from "../../apis/weather";
+
+import * as actionTypes from "../action-types";
+
+export const getWeather = (lat, long) => async (dispatch) => {
+  //temporarily hardcoded for single city
+  // to be implemented: pass lat, long to get weather of searched city
+  const response = await axios.get("/forecast?lat=51.05&lon=-114.07");
```

Comment:
create `apiEndpoint.js` file constants folder like the way defined above

Reply:
also, create a services folder and create a weather.js file. Inside the services/weather.js file create a method `getWeather` method